expression/util.go

```diff
@@ -1794,3 +1798,51 @@ func ExprsToStringsForDisplay(exprs []Expression) []string {
 	}
 	return strs
 }
+
+// SortAndGenMD5HashForCNFExprs sorts cnf exprs by their hash codes
```

Comment:
Does this sort along CNF and DNF recursively?
Initially, sorting along CNF at the outermost level is good enough.

Reply:
Now it only sorts standard CNF expression:   (A or B or C) and (D or E or F) and (G or H or I), **and** expression inside DNF will not be sorted recursively.

Reply:
It is more general to sort the expression along CNF and DNF recursively, I'll change it.